notebooks/applications/spark/README.md

```diff
@@ -90,43 +90,6 @@ Steps to create a notebook server are found [here](https://github.com/KubeSoup/d
       - [Running on Kubernetes Configuration](https://spark.apache.org/docs/latest/running-on-kubernetes.html#configuration) - attirbutes specific to kubernetes
       - [Spark Integration with Amazon Web Services](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html) - attributes to configuring access to S3 and other AWS related services
 
-
-## Optional: Verification of Modifications:
-
-Run the following commands, where the expected output is shown.
-
-1. `kubectl get svc $NOTEBOOK_NAME -o yaml`:
-
-    ```
-    spec:
-      ports:
-      - name: driver
-        port: 2222
-        protocol: TCP
-        targetPort: 2222
-      - name: blockmanager
-        port: 7078
-        protocol: TCP
-        targetPort: 7078
-      - name: spark-ui
-        port: 4040
-        protocol: TCP
-        targetPort: 4040
-    ```
-
-    If it is not the case, try to delete the svc one time `kubectl delete svc $NOTEBOOK_NAME` (you will loose the connection to the notebook for a while - refresh the page).
-
-2. `kubectl get StatefulSet $NOTEBOOK_NAME -o yaml`:
-
-    ```
-    spec:
-      template:
-        metadata:
-          annotations:
-            traffic.sidecar.istio.io/excludeInboundPorts: "7078"
-    ```
-
-
```

Comment:
I am not sure about this part, as it helps the users to debug if the spark automation is setup correctly.

Reply:
Hey Salil,

I generally agree that it help debugging issues.

However, a user shouldn't need to debug issues. Spark should just work. If it doesn't work, one of us would probably debug it using Lens and NOT kubectl. :D

Furthermore, this part is automated in a way that is not visible to the user. I don't see why the user should be able to debug it.

This is at least my motivation. If you would like to keep it for other purposes, I could put this in the internal docs.

What do you think?

Reply:
Yes, keeping it on internal docs is also nice. I agree.